Update chromedriver and selenium to fix failing tests

chrisgarrity opened the pull request:
Chromedriver 81 is available, but gets an error running the install script. Using chromedriver 80 instead. It fixed the tests for me - hopefuly it does for travis too.

@ericrosenbaum I'm tagging you for review since you're next on the deploy schedule and this needs to be fixed by next week. We haven't updated translations for 2 deploy cycles due to the tests failing. 

Testing:
If the travis build of the PR doesn't fail, then it's fixed.

chrisgarrity committed:
Update chromedriver and selenium to fix failing tests

Pinned at chromedriver 80.0.1 because 80.0.2 stopped supporting node 8. (Thanks @apple502j for pointing at the issue). If you try to install chromedriver with node 8, the install script produces an error.

apple502j commented on package.json:
```diff
@@ -30,7 +30,7 @@
   "devDependencies": {
     "async": "2.6.0",
-    "chromedriver": "^75.0",
+    "chromedriver": "^80.0.0",
```
```suggestion
    "chromedriver": "80.0.1",
```

Node v8 support was dropped on `80.0.2`:  giggio/node-chromedriver#261

chrisgarrity replied:
👍

apple502j commented:
@chrisgarrity It seems like we need to either use `80.0.1` or bump Node version to at least 10.

chrisgarrity commented:
@apple502j thanks for the pointer to the chromedriver issue - that explains a lot.

chrisgarrity committed:
Update chromedriver to latest

In a separate PR we updated the travis config to Node 12, so the installation should work for testing

chrisgarrity commented:
@rschamp since this was fixed by upgrading node to version 12, I'm closing this PR. It might still be a good idea to change the chromedriver version to `^81.0` at some point
